content/en/security/guide/tls_deprecation_1_2.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Deprecation notice for TLS version < 1.2
+kind: guide
+---
+
+
+## Overview
+
+Transport Layer Security (TLS) is a critical security protocol used to protect web traffic. It provides confidentiality and integrity of data in transit between clients and servers exchanging information. Datadog is disabling support for older versions of TLS, below 1.2 (SSLv3, TLS v1.0, TLS v1.2) across public facing Datadog applications, beginning March 31, 2022. If you use unsupported clients to connect to Datadog after the older protocols are disabled, you will receive connection error messages.
+
+### Reason for deprecation
+
+These protocols are being deprecated to ensure that customers connect to Datadog using secure connection channels. This is in accordance with a decision from the Internet Engineering Task Force (IETF) to deprecate these protocols as of March 25, 2021. ([https://datatracker.ietf.org/doc/rfc8996/][1])
+
+## Browser support
+
+Modern browsers have had support for TLS 1.2 for a while. See the "Can I use..." [compatibility matrix][2] to determine if your specific browser and version are affected.
+
+## Language support
+
```

Comment:
```suggestion
### Openssl

OpenSSL is a library for general-purpose cryptography and secure communication used by many other tools like Python, Rub, PHP, Curl... TLS v 1.2 has been supported since OpenSSL 1.0.1, see https://www.openssl.org/news/changelog.html#openssl-101.
```